Ginger/GingerCore/ALM/RQM/ExportToRQM.cs

```diff
@@ -211,9 +212,10 @@ public bool ExportExecutionDetailsToRQM(BusinessFlow businessFlow, ref string re
                                 return false;
                             }
                         }
-                        catch
+                        catch(Exception ex)
                         {
                             Reporter.ToLog(eLogLevel.ERROR, $"Failed to Update Execution Record Results for {businessFlow.Name} and testplan {bfExportedID}");
```

Comment:
Keep only one log with Error level and exception as a parameter to it

Reply:
updated